src/client/common/components/base-network-view/config/index.js

```diff
@@ -1,6 +1,6 @@
 const h = require('react-hyperscript');
 
-const { NetworkInfoMenu, FileDownloadMenu } = require('./menus');
+const { NetworkInfoMenu, FileDownloadMenu, InteractionsFilterMenu} = require('./menus');
```

Comment:
See https://github.com/PathwayCommons/app-ui/blob/master/src/client/features/paint/index.js#L31

for how to inject feature specific menus into the base network component